### ISSUE TITLE
Resizing the window crashes samples

### ISSUE THREAD

**Issue by ConorLPBoyle** [GPUOpen-LibrariesAndSDKs/V-EZ#8]
Resizing the window of MSVC x64 Win10 compiled samples sometimes results in a properly resized framebuffer, but often crashes the program. Release builds handle resizing correctly more often, but eventually crash as well. In visual studio it triggers a breakpoint at [SimpleQuad.cpp line 98](https://github.com/GPUOpen-LibrariesAndSDKs/V-EZ/blob/master/Samples/SimpleQuad/SimpleQuad.cpp#L98): `Unhandled exception at 0x00007FFF5A614008 in SimpleQuadD.exe: Microsoft C++ exception: std::bad_function_call at memory location 0x0000004E773AE810.`

Some of the validation warnings presented in debug builds:
```
Name: GeForce GTX 1050
Type: DISCRETE_GPU
MEM(WARNING): Object: 0x12 | vkGetFenceStatus called for fence 0x12 which has not been submitted on a Queue or during acquire next image.
ObjectTracker(ERROR): Object: 0x177a5bcc288 | Invalid CommandBuffer Object 0x177a5bcc288. The spec valid usage text states 'pCommandBuffers must be a valid pointer to an array of commandBufferCount VkCommandBuffer handles, each element of which must either be a valid handle or NULL' (https://www.khronos.org/registry/vulkan/specs/1.0/html/vkspec.html#VUID-vkFreeCommandBuffers-pCommandBuffers-00048)
ObjectTracker(ERROR): Object: 0x177a5bcc288 | Unable to remove CommandBuffer obj 0x177a5bcc288. Was it created? Has it already been destroyed?
ObjectTracker(ERROR): Object: 0x177c8f6f6d8 | Invalid CommandBuffer Object 0x177c8f6f6d8. The spec valid usage text states 'pCommandBuffers must be a valid pointer to an array of commandBufferCount VkCommandBuffer handles, each element of which must either be a valid handle or NULL' (https://www.khronos.org/registry/vulkan/specs/1.0/html/vkspec.html#VUID-vkFreeCommandBuffers-pCommandBuffers-00048)
ObjectTracker(ERROR): Object: 0x177c8f6f6d8 | Unable to remove CommandBuffer obj 0x177c8f6f6d8. Was it created? Has it already been destroyed?
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.

...

MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
THREADING(ERROR): Object: 0x8 | THREADING ERROR : object of type VkFence is simultaneously used in thread 22072 and thread 6244
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.

...

MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
THREADING(ERROR): Object: 0x8 | THREADING ERROR : object of type VkFence is simultaneously used in thread 22072 and thread 6244
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
THREADING(ERROR): Object: 0x8 | THREADING ERROR : object of type VkFence is simultaneously used in thread 6244 and thread 22072
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.

...

MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
THREADING(ERROR): Object: 0x8 | THREADING ERROR : object of type VkFence is simultaneously used in thread 22072 and thread 6244
THREADING(ERROR): Object: 0x8 | THREADING ERROR : object of type VkFence is simultaneously used in thread 22072 and thread 6244
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
```

```
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
MEM(WARNING): Object: 0x8 | vkGetFenceStatus called for fence 0x8 which has not been submitted on a Queue or during acquire next image.
MEM(WARNING): Object: 0x22 | vkGetFenceStatus called for fence 0x22 which has not been submitted on a Queue or during acquire next image.
```

**Comment by soconne** [comment]
Thank you for reporting this, I will investigate.

**Comment by soconne** [comment]
This is now fixed in the v1.0.2 release.
